openapi.yaml

```diff
@@ -0,0 +1,69 @@
+swagger: '2.0'
+info:
+  version: '1.0.0'
+  title: Fulcio
+schemes: [http, https]
+host: fulcio.sigstore.dev
+basePath: /v1
+consumes: [application/json]
+produces: [application/json]
+securityDefinitions:
+  Bearer:
```

Comment:
I think this should be OAuth2

Reply:
OAuth2 would be the Authentication Server {keycloak,kong}, I expect this would instead be a scoped JWT token?

Reply:
what's view on this now @bobcallaway ?